src/components/HomePage/Portfolio/index.tsx

```diff
@@ -37,31 +31,18 @@ const Portfolio: React.FC = () => {
         <h1 className="text-4xl font-bold text-center mb-12">
           Explore My Photography Work
         </h1>
-        <div className="flex justify-center space-x-4 mb-6">
+      </div>
+      <div className="container mx-auto">
+        <div className="flex justify-end space-x-4 mb-6">
           <Button className="p-2 rounded-full">{"<"}</Button>
           <Button className="p-2 rounded-full">{">"}</Button>
         </div>
-        <div className="grid md:grid-cols-3 gap-6">
-          {projects.map((project, index) => (
-            <Link
-              href={project.link}
-              key={index}
-              className="flex group cursor-pointer w-96 h-96 bg-cover bg-center rounded-lg overflow-hidden relative"
-              style={{ backgroundImage: `url(${project.image})` }}
-            >
-              <div className="flex place-self-end bottom-4 left-4 flex-col relative z-10">
-                <h2 className="text-xl font-semibold">{project.title}</h2>
-                <p className="text-sm">{project.date}</p>
-                <p className="mt-2 text-sm font-semibold">VIEW PROJECT</p>
-              </div>
-            </Link>
-          ))}
-        </div>
-        <div className="text-center mt-8">
-          <Link href="/all-works">
-            <Button className="px-6 py-3 rounded-lg">View All Works →</Button>
-          </Link>
-        </div>
+        <ProjectList projects={projects} />
+      </div>
+      <div className="text-center mt-8">
```

Comment:
do you need text center in this div?
also I'm not sure that you need div here - you may just use Link component, but I'm not sure if it has display: block in it

Reply:
div for button